<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stockdetails</class>
 <widget class="QWidget" name="stockdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>股票详情</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>30</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>股票ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ID">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>股票数量:</string>
   </property>
  </widget>
  <widget class="QLabel" name="number">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>539</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起点股价:</string>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>price</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 255, 249);
border-radius:25px;    /*设置圆角半径 */
font:bold 14px;		/*设置按钮文字和大小*/</string>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出价 元/股</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>预入股数</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(255, 63, 25);
border-color: rgb(138, 255, 255);
border-radius:25px;    /*设置圆角半径 */
padding:2px 4px;  /*QFrame边框与内部其它部件的距离*/
color:white;		/*字体颜色*/
font:bold 14px;		/*设置按钮文字和大小*/
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
